dockerfiles/cli.Dockerfile

```diff
@@ -55,3 +55,5 @@ RUN mkdir -p $AKAMAI_CLI_HOME/.akamai-cli ${AKAMAI_CLI_CACHE_PATH}
 COPY --from=builder /akamai.upx /bin/akamai
 
 ADD files/akamai-cli-config ${AKAMAI_CLI_HOME}/.akamai-cli/config
+
+RUN chmod -R a+rwx ${AKAMAI_CLI_HOME}
```

Comment:
is there any need or benefit to set the whole directory tree to 0777 instead of "/cli/.akamai-cli/cache" only? or in your opinion it's safe bet (as containers are volatile anyway..)?

Reply:
The specific issue prompting this was on perm problems around /cli/.akamai-cli/config.
It seems to me that the easiest is to whitelist all folders that can be mutated by packages in Jenkins context.
I also confess that I don't have much background on the security impact of such a change, so this is really a break-fix thing rather than a really well thought out approach.

Reply:
Thanks for the details!